pavement.py

```diff
@@ -73,9 +73,22 @@ def make_venv():
     bootstrap_venv(__here__ / path('env'), 'sideboard')
     develop_sideboard()
 
+def install_pip_requirements_in_dir(dir_of_requirements_txt):
+    path_to_pip = __here__ / path('env/bin/pip3')
+    sh('{pip} install -e {dir_of_requirements_txt}'
+        .format(
+            pip=path_to_pip,
+            dir_of_requirements_txt=dir_of_requirements_txt))
+
+def run_setup_py(path):
+    sh('cd {path} && {python_path} {setup_path} develop'
+        .format(
+            path=path,
+            python_path=sys.executable,
+            setup_path=join(path, 'setup.py')))
+
```

Comment:
actually, can't remove it because it's used by another function

when we have docker going for real, we should either:
1) move a lot of setup functionality into pavement.py, like creating the venv and doing pip stuff
2) move it all out and into docker setup scripts

I think at the time we continue doing docker stuff we should revisit everything in this file.